### PR TITLE
Alteration filing changes

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#4168

*Description of changes:*
Added optional attributes related to
1. Resolution/Court Order Date
2. Has Pre-existing company provisions
3. Name Translations

to the conversion schema

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

File: tests/unit/test_alteration.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    if errors:
+        for err in errors:
+            print(err.message)
+    print(errors)
+
+    assert is_valid
+
+
+def test_validate_invalid_alteration_with_no_required_elements():
+    """Assert not valid if name request node is not present."""
```
Comment (thorwolpert) [inline tests/unit/test_alteration.py:52]: doesn't seem to match the test

File: tests/unit/test_alteration.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    if errors:
+        for err in errors:
+            print(err.message)
+    print(errors)
+
+    assert not is_valid
+
+
+def test_validate_invalid_name_translation_alteration():
+    """Assert not valid if legalType is not an accepted type."""
```
Comment (thorwolpert) [inline tests/unit/test_alteration.py:100]: description does not match test

File: tests/unit/test_alteration.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    if errors:
+        for err in errors:
+            print(err.message)
+    print(errors)
+
+    assert not is_valid
+
+
+def test_validate_invalid_share_structure_alteration():
+    """Assert not valid if legalType is not an accepted type."""
```
Comment (thorwolpert) [inline tests/unit/test_alteration.py:116]: description does not match test

**Comment by lekshmimallika-aot** [comment]
> bumping the version?

Updated :)

**Pull request merged**